docs/UserMenu.md

```diff
@@ -55,17 +55,19 @@ const avatar = <Avatar><SendIcon/></Avatar>;
-| ------------ | ------------------------------------------------ | ------------------ | -------- | ------- |
-| avatar       | MUI Avatar that displays                         | `Avatar`           | yes      |         |
-| classes      | Style overrides                                  | `UserMenuClasses`  | no       |         |
-| menu         | Custom MUI Menu displayed when Avatar is clicked | Material-UI `Menu` | no       |         |
-| menuGroups   | Groups of menu items that display                | `UserMenuGroups[]` | no       |         |
-| menuSubtitle | Subtitle shown when menu is open                 | `string`           | no       |         |
-| menuTitle    | Title shown when menu is open                    | `string`           | no       |         |
-| MenuProps    | Property overrides for the MUI Menu              | `MenuProps`        | no       |         |
-| onClose      | Function called when the menu is closed          | `Function`         | no       |         |
-| onOpen       | Function called when the menu is opened          | `Function`         | no       |         |
+| Prop Name          | Description                                                               | Type               | Required | Default |
+| ------------------ | ------------------------------------------------------------------------- | ------------------ | -------- | ------- |
+| avatar             | MUI Avatar that displays                                                  | `Avatar`           | yes      |         |
+| classes            | Style overrides                                                           | `UserMenuClasses`  | no       |         |
+| disableBottomSheet | Disable responsive bottom sheet menu                                      | `boolean`          | no       | false   |
+| menu               | Custom MUI Menu displayed when Avatar is clicked                          | Material-UI `Menu` | no       |         |
+| menuGroups         | Groups of menu items that display                                         | `UserMenuGroups[]` | no       |         |
+| menuSubtitle       | Subtitle shown when menu is open                                          | `string`           | no       |         |
+| menuTitle          | Title shown when menu is open                                             | `string`           | no       |         |
+| MenuProps          | Property overrides for the MUI Menu                                       | `MenuProps`        | no       |         |
+| onClose            | Function called when the menu is closed                                   | `Function`         | no       |         |
+| onOpen             | Function called when the menu is opened                                   | `Function`         | no       |         |
+| useBottomSheetAt   | Window pixel width at which the responsive bottom sheet menu is triggered | `number`           | no       | 600     |
 
 </div>
```

Comment:
I would suggest adding a note somewhere that says if you pass in a custom menu using the `menu` prop, then the responsiveness no longer applies.

Reply:
I would do that in the initial description paragraph (you can add a blip about how it switches between menu and bottomsheet, but if you supply your own menu you have to do it yourself).